### ISSUE TITLE
Old datasets cannot be edited due to not having ORCID login at that time

### ISSUE THREAD

**Issue by mahirst** [datadryad/dryad-product-roadmap#3648]
I've gotten a few help desk cases in the last week where I needed to go into an old (2017, 2018) dataset and update it (an author wants to link his ORCID, and a journal wants to add the article DOI); however, I get the error message to add the submitting author's email, so I can't make any of the requested edits to the dataset. 

Audrey said: In these, the issue is usually actually that they are so old that they have main users who did not use the ORCID login and have no ORCID ID. The ORCID is the only connecting ID between the author list (StashDatacite::Authors) and the user account (StashEngine::User), so the true issue here is that without a matching ORCID between the author list and the user account, the system cannot identify which is the “submitting author” in order to ensure they have an email address listed

#### DOIs affected:

**10.5061/dryad.k57430s**
Resource ID: 27584
Fernando Alda; ORCID 0000-0002-6858-9566; email: fernando.alda@csic.es
This author just wants to link his ORCID to the dataset. I'm not sure who the submitting author was.

**10.5061/dryad.6n8kf**
Resource ID:
Looks like the submitter is Francis Choi, ORCID 0000-0003-4389-8087; email: fmp.choi@gmail.com
In this case, the journal wants to link the published article DOI https://doi.org/10.1093/conphys/coae025

**Comment by ahamelers** [comment]
For the first one, the owner was not an author at all and had a Dryad email address (probably a curator at the time!). I made Fernando the owner and linked up his email address.

For the second, I merged the two Francis Choi users we had in our system, one which had only an email and the other only an ORCID. Then I made sure those were reflected in the author list. Just a note for this one @mahirst , it already has a primary article listed, which seems totally correct—I'm guessing this other DOI is just a 'related' article?
